protocol/src/mercury.rs

```diff
@@ -50,17 +50,17 @@ impl MercuryMultiGetRequest {
 
     // repeated .MercuryRequest request = 1;
```

Comment:
This file is generated automatically from the protobuf files. It shouldn't be modified.

Reply:
Thanks for the explanation. I dropped the commit.